packages/ketcher-react/src/script/editor/tool/template.ts

```diff
@@ -506,7 +504,6 @@ function addSaltsAndSolventsOnCanvasWithoutMerge(
   const [action] = fromTemplateOnCanvas(restruct, template, dragCtx.xy0, 0)
   editor.update(action)
   editor.selection(null)
-  editor.hover(null)
```

Comment:
I'm really not sure this is a good idea to remove hover resets from all tools.
They are there for some reason, I guess.
Is it expected that hover will never disappear for all other tools (like here, for example)?

Reply:
reworked